components/expand-collapse/expand-collapse-content.js

```diff
@@ -130,6 +128,7 @@ class ExpandCollapseContent extends LitElement {
 					this._state = states.EXPANDING;
 					const content = this.shadowRoot && this.shadowRoot.querySelector('.d2l-expand-collapse-content-inner');
 					if (content) this._height = `${content.scrollHeight}px`;
```

Comment:
This change is "ok" because the `this._height` will not be applied until Lit's async lifecycle runs. If this synchronously updated the DOM, the line that immediately follows would force the browser to recompute layout in order to obtain a fresh `scrollHeight`, even if the property that invalidated the layout wasn't height related. Just something to be cautious of... so in general when obtaining DOM measurements that are used more than once, I usually obtain them once and assign them to a constant or variable.